apps/web/src/utils/notifications.tsx

```diff
@@ -1,3 +1,4 @@
-import { errorMessage, successMessage } from '@novu/design-system';
+import { errorMessage as errorMessageRoot, successMessage as successMessageRoot } from '@novu/design-system';
 
-export { errorMessage, successMessage };
+export const errorMessage = errorMessageRoot;
+export const successMessage = successMessageRoot;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Why make this abstraction here?‏